### DISCUSSION TITLE
Paketo Project Specification

### DISCUSSION THREAD

**Discussion by ForestEckhardt** [paketo-buildpacks/feedback#17] [Ideas]
Recently we have merged some RFCs that layout some specifics for the Paketo Project such as:
* [SemVer RFC](https://github.com/paketo-buildpacks/rfcs/blob/main/text/0029-semantic-versioning.md)
* [Bill of Materials RFC](https://github.com/paketo-buildpacks/rfcs/blob/main/text/0033-bill-of-materials.md)

We have also had discussions about layout of set of guidelines as to what is required to be a core Paketo buildpack. These things feel like they could benefit from being part large Paketo project specification.

I am curious to hear peoples thoughts on this idea as well as some other things that they may want added to a Paketo specification.

An example of an addition might be something like respect `build` and `launch` flags inside of build plans across the whole project.

**Comment by fg-j** [comment]
@ekcasey made an interesting point about documentation vs. specification in the CNB working group meeting the other day that I think is also relevant here. I'll hold off on paraphrasing if you want to chime in here, Emily.
